app/views/layouts/_action_menu_base.html.erb

```diff
@@ -28,6 +28,12 @@ See doc/COPYRIGHT.rdoc for more details.
 ++#%>
 
 <% action_menu_key = action_menu_key ? action_menu_key : :action_menu_main %>
+<%
+   # TODO FIXME OMG! this is the ugliest hack I ever performed
+   # We need to hide the clearfix in the wiki to avoid additional spacing in the wiki
+   # THIS HACK NEEDS TO BE REPLACED BY AN ENGINEERS SOLUTION!
+   @no_clearfix ||= false
```

Comment:
You could remove `@no_clearfix` and just check for 

```
@no_clearfix.present?
```

in the condition AFAIK.